about.html

```diff
@@ -0,0 +1,138 @@
+                next level. You can stop learning. If you do then fashion will move past you. 
+                We are also having different session with different sessoned professionals and designers all over the country.
+                Don't miss out on this opportunity to take your career to the next level.
+            </p>
+            <div class="bg-white-color">Please contact us more questions in Globbery fashion training 2022</div>
+            <div class="bg-white-color">services@globerry.org</div>
+        </div>
+    </section>
+
+
+    <section class="abt-session">
+        <div class="container">
+            <div class="row">
+                
+                <h6> Globbery Training and Fashion School 2022</h6>
+                <div class="col-12 exerpt-content  mx-auto">
+                   <p>This training is a once in a life time. Is a training that will make you a better person.
+                       Don't miss out on this opportunity of making it. Our programme will start with intense
+                       training session and continue to ensure that everything is going fine.
+                   </p>
+                 </div>
+
+                 <div class="col-12">
+                     <img src="./images/logo6.png" class="making-border img-fluid" alt="Globerry Logo"/>
+                 </div>
+                 
+            </div>     
+        </div>
+    </section>
+
+
+    <section class="past-trainings">
+        <h3 class="text-center">See Past Globery Trainings</h3>
+        <p>Take a look at the two(2) Globerry Couture which took place at Lameridian Hotel, Igoja Lagos.</p>
+        
+        <div class="fashion-events">
+            <div class="wrapper">
+                <div class="content">
+                 <p>2012</p>
+                 <div>Designers Trend for Fashion Longe</div>
+                </div>
+            </div>
+
+            <div class="wrapper2">
+                <div class="content2">
+                 <p>2013</p>
+                 <div>Training Session with Indian Tailors</div>
+                </div>
+            </div>
+            <div class="wrapper3">
+                <div class="content3">
+                 <p>2015</p>
+                 <div>Trends with Globbery</div>
+                </div>
+            </div>
+        </div>
+      
+    </section>
+
+
+    <footer class="footer_section make-dark">
+        <div class="container">
+            <div class="row">
+                <div class="col-4">
+                    <a href=""><img src="./images/globerry.PNG" class="top-img" alt="Globerry Logo"/></a>
+                   
+                </div>
+                <div class="col-8">
+                    <p>2022 Globerry Couture. Some rights reserved</p>
+                    <p>We are the best there is, we can make you better and get closer to your dreams.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] After you added the logo, it now overlaps with the text aligned to the right. Please fix the logo not to exceed the width of its container in order to get rid of that overlap 👍 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/169617399-1a3123b4-c0ac-418c-8d6a-2e1d47e6818e.png)
### Guideline
![image](https://user-images.githubusercontent.com/46839548/169617457-c2518352-6b2a-4ca4-a399-de661a8a6fed.png)